File: src/drivetrain/HolonomicDrivetrain.cpp
// Copyright (c) TrajoptLib contributors

#include "drivetrain/HolonomicDrivetrain.h"

#include "drivetrain/Drivetrain.h"
#include "obstacle/Obstacle.h"

namespace trajopt {

HolonomicDrivetrain::HolonomicDrivetrain(double mass, double momentOfInertia)
    : Drivetrain(mass, momentOfInertia) {}
}  // namespace trajopt

File: java/src/main/java/org/sleipnirgroup/trajopt/HolonomicDrivetrain.java
// Copyright (c) TrajoptLib contributors

package org.sleipnirgroup.trajopt;

public abstract class HolonomicDrivetrain extends Drivetrain {
    protected HolonomicDrivetrain(double mass, double momentOfInertia, Obstacle bumpers) throws NullPointerException {
        super(mass, momentOfInertia, bumpers);
    }
}
